configure

```diff
@@ -323,10 +323,11 @@ valopt target "${CFG_HOST}" "GNUs ./configure syntax LLVM target triples"
 
 valopt localstatedir "/var/lib" "local state directory"
 valopt sysconfdir "/etc" "install system configuration files"
-valopt datadir "${CFG_PREFIX}/share" "install data"
-valopt infodir "${CFG_PREFIX}/share/info" "install additional info"
-valopt mandir "${CFG_PREFIX}/share/man" "install man pages in PATH"
-valopt libdir "${CFG_PREFIX}/lib" "install libraries"
+valopt datadir "share" "install data (relative to prefix)"
```

Comment:
But now that I read my logic I see where it won't work. Need to keep it how it was and before saving the value to `config.mk` strip `CFG_PREFIX` from the path. I'll redo the patches that way.